api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiProperties.java

```diff
@@ -61,4 +63,24 @@ public boolean isProbabilisticMatchingEnabled() {
             return false;
         }
     }
+
+    public List<String> getPixIdentifierTypeUuidList() {
+        String uuidListPropertyName = RegistrationCoreConstants.GP_MPI_PIX_IDENTTIFIER_TYPE_UUID_LIST;
```

Comment:
I still see the redundant assignment of sting constants to variables in both methods